pytoniq/contract/wallets/highload_v3.py

```diff
@@ -0,0 +1,143 @@
+
+    @classmethod
+    async def create(cls, provider: LiteClientLike, wc: int = 0, wallet_id: typing.Optional[int] = None, timeout: typing.Optional[int] = None):
+        """
+        :param provider: provider
+        :param wc: wallet workchain
+        :param wallet_id: subwallet_id
+        :param timeout: timeout
+        :return: mnemonics and Wallet instance of provided version
+        """
+        mnemo = mnemonic_new(24)
+        return mnemo, await cls.from_mnemonic(provider=provider, mnemonics=mnemo, wc=wc, wallet_id=wallet_id, timeout=timeout)
+
+    @staticmethod
+    def raw_create_transfer_msg(private_key: bytes, wallet_id: int, sendmode: int, created_at: int, timeout: int, message_to_send: WalletMessage, query_id: HighloadQueryId = 0) -> Cell:
+        signing_message = Builder() \
+            .store_uint(wallet_id, 32) \
+            .store_ref(message_to_send) \
+            .store_uint(sendmode, 8) \
+            .store_uint(query_id.shift, 13) \
+            .store_uint(query_id.bit_number, 10) \
+            .store_uint(created_at, 64) \
+            .store_uint(timeout, 22) \
+        .end_cell()
+        signature = sign_message(signing_message.hash, private_key)
+        return Builder() \
+            .store_bytes(signature) \
+            .store_cell(signing_message) \
+        .end_cell()
+
+    async def raw_transfer(self, sendmode: int, created_at: int, timeout: int, msg: WalletMessage, query_id: HighloadQueryId = 0):
+        """
+        :param sendmode: sendmode
+        :param created_at: created at
+        :param timeout: timeout
+        :param msg: WalletMessage. to create one call create_wallet_internal_message method
+        :param query_id: query id
+        """
+        if 'private_key' not in self.__dict__:
+            raise WalletError('must specify wallet private key!')
+
+        transfer_msg = self.raw_create_transfer_msg(private_key=self.private_key, wallet_id=self.wallet_id, 
+                                                    sendmode=sendmode, created_at=created_at, timeout=timeout,
+                                                    message_to_send=msg, query_id=query_id)
+
+        return await self.send_external(body=transfer_msg)
+
+    async def transfer(self, destination: typing.Union[Address, str], amount: int, sendmode: int, created_at: int, timeout: int, body: Cell = Cell.empty(), state_init: StateInit = None):
+        if isinstance(destination, str):
+            destination = Address(destination)
+
+            result_msg = self.create_wallet_internal_message(destination=destination, value=amount, body=body, state_init=state_init)
+        return await self.raw_transfer(sendmode=sendmode, created_at=created_at, timeout=timeout, msg=result_msg)
+
+    async def send_init_external(self, sendmode: int, created_at: int, timeout: int, message_to_send: WalletMessage):
```

Comment:
In my opinion, this is good place to set default values.
For example, I don't think many users will really understand meaning of `timeout` var and how it works. Let's advice some value.